src/main/webapp/community.html

```diff
@@ -214,7 +214,7 @@ <h4 class="pb-3 mb-4">Apply pull requests</h4>
                         <code>username  git@github.com:username/karaf.git (fetch)</code><br>
                         <code>username  git@github.com:username/karaf.git (push)</code>
                     </li>
-                    
```

Comment:
Please avoid extra change as this one, it creates noise in the diff.

Reply:
Up to you, that's not a big deal for karaf-site (could be problematic for code).

Reply:
Agree for code. I think we could keep the clean here.